mbrl/models/gaussian_mlp.py

```diff
@@ -56,14 +56,16 @@ class GaussianMLP(Ensemble):
                           input -h1-> -h2-> -l3-> output).
         ensemble_size (int): the number of members in the ensemble. Defaults to 1.
         hid_size (int): the size of the hidden layers (e.g., size of h1 and h2 in the graph above).
-        use_silu (bool): if ``True``, hidden layers will use SiLU activations, otherwise
+        use_silu (bool) (depricated, use activation_cls instead): if ``True``, hidden layers will use SiLU activations, otherwise
                          ReLU activations will be used. Defaults to ``False``.
         deterministic (bool): if ``True``, the model will be trained using MSE loss and no
             logvar prediction will be done. Defaults to ``False``.
         propagation_method (str, optional): the uncertainty propagation method to use (see
             above). Defaults to ``None``.
         learn_logvar_bounds (bool): if ``True``, the logvar bounds will be learned, otherwise
             they will be constant. Defaults to ``False``.
+        activation_name (str, optional): name of the activation function to use from torch.nn. Defaults to ``ReLU``.
+        activation_params (list, optinal): list of parameteres, if required, for the chosen activation as per the definition in torch.nn. Defaults to None. (e.g., activation_name = ``Threshold``, activation_params = [0.5,10] (threshold, value))
```

Comment:
Rather than pass the parameters, let's take advantage of Hydra here. All of the activation information (including the class name) can be passed as a dictionary and instantiated as described in [this](https://github.com/facebookresearch/mbrl-lib/pull/124#discussion_r697859674) comment, following the example of `BasicEnsemble`.